gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -7,17 +7,34 @@
 module PostgresHaAdmin
   class FailoverMonitor
     FAILOVER_ATTEMPTS = 10
-    DB_CONNECTED_CHECK_FREQUENCY = 300
+    DB_CHECK_FREQUENCY = 300
     FAILOVER_CHECK_FREQUENCY = 60
+    attr_accessor :failover_attempts, :db_check_frequency,
+                  :failover_check_frequency
```

Comment:
This can move back to one line. I think that change was made because of the faulty bot comments.